eap74-rhel8-payg-multivm/src/test/scripts/gen-parameters.sh

```diff
@@ -2,7 +2,7 @@
 set -Eeuo pipefail
 
 #read arguments from stdin
-read parametersPath gitUserName testbranchName location vmName asName adminUsername password enableLb numberOfInstances operatingMode virtualNetworkResourceGroupName bootStorageAccountName storageAccountResourceGroupName jbossEAPUserName jbossEAPPassword rhsmUserName rhsmPassword rhsmPoolEAP
```

Comment:
This will have to change.  See https://devdiv.visualstudio.com/DevDiv/_workitems/edit/1620889 .